package.json

```diff
@@ -30,9 +30,7 @@
     "watch": "lerna run --parallel watch",
     "watch:exe": "cd zowex && cargo install cargo-watch && cargo watch -x build -x clippy -x test",
     "watch:test": "jest --watch",
-    "doc:clean": "rimraf docs/CLIReadme.md",
-    "doc:generate": "npm run doc:clean && gulp doc",
-    "generateCleanTypedoc": "npm run typedoc && gulp cleanTypedoc",
+    "generateCleanTypedoc": "npm run typedoc",
```

Comment:
should we remove this script too since it's basically just calling typedoc 😋

Reply:
I didn't want to just in case it was being called somewhere, but we can if you prefer.